src/terminal/input/terminalInput.cpp

```diff
@@ -494,21 +494,18 @@ bool TerminalInput::HandleKey(const IInputEvent* const pInEvent)
     // for the 5 least significant ones to be zeroed out.
     if (keyEvent.IsAltPressed() && keyEvent.IsCtrlPressed())
     {
-        auto ch = keyEvent.GetCharData();
-        if (ch == UNICODE_NULL)
-        {
-            // For Alt+Ctrl+Key messages GetCharData() returns 0.
-            // -> Get the char from the virtual key.
-            ch = LOWORD(MapVirtualKeyW(keyEvent.GetVirtualKeyCode(), MAPVK_VK_TO_CHAR));
-        }
-        if (ch == UNICODE_SPACE)
+        auto character = _CharacterFromAltCtrlKeyEvent(keyEvent);
+        if (character)

```

Comment:
```suggestion
        if (character.has_value())
```

I _know_ that these do the same thing, but I find the second version much more readable, because I initially thought the error case was `UNICODE_NULL`